src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started working");
```

Comment:
The message 'Excavator started working' could be more informative by specifying that it is an Excavator machine that is working, to align with the checklist requirement for informative messages.